This reverts the commit that caused ibus-anthy to stop working after version 1.5.17.
See details: https://github.com/ibus/ibus-anthy/commit/14b2759772fb1adc67453108e958bf0bf181ce1d

diff --git a/engine/python2/main.py b/engine/python2/main.py
index c1e6a2f..c366293 100644
--- a/engine/python2/main.py
+++ b/engine/python2/main.py
@@ -41,10 +41,6 @@ GLib.set_prgname('ibus-engine-anthy')
 from gi.repository import IBus
 
 import _config as config
-
-# Need to call IBus.init() before IBus.EngineSimple() is loaded.
-# factory -> engine -> IBus.EngineSimple
-IBus.init()
 import factory
 
 class IMApp:
@@ -183,6 +179,7 @@ def main():
             print >> sys.stderr, 'Unknown argument: %s' % o
             print_help(sys.stderr, 1)
 
+    IBus.init()
     if daemonize:
         if os.fork():
             sys.exit()
diff --git a/engine/python3/main.py b/engine/python3/main.py
index 05836a6..8665ae0 100644
--- a/engine/python3/main.py
+++ b/engine/python3/main.py
@@ -41,10 +41,6 @@ GLib.set_prgname('ibus-engine-anthy')
 from gi.repository import IBus
 
 import _config as config
-
-# Need to call IBus.init() before IBus.EngineSimple() is loaded.
-# factory -> engine -> IBus.EngineSimple
-IBus.init()
 import factory
 
 class IMApp:
@@ -181,6 +177,7 @@ def main():
             print('Unknown argument: %s' % o, file=sys.stderr)
             print_help(sys.stderr, 1)
 
+    IBus.init()
     if daemonize:
         if os.fork():
             sys.exit()
